new/mip_process_and_guidelines.md

```diff
@@ -0,0 +1,210 @@
+- Simple Summary - “If you can’t explain it simply, you don’t understand it well enough.” Provide a simplified and layman-accessible explanation of the MIP.
+- Abstract - a short (~200 word) description of the technical issue being addressed.
+- Motivation (*optional) - The motivation is critical for MIPs that want to change the Mercury protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the MIP solves. MIP submissions without sufficient motivation may be rejected outright.
+- Specification - The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Mercury platforms, [and others](https://github.com/libertaria-project/wiki/wiki/Clients).
+- Rationale - The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.
+- Backwards Compatibility - All MIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The MIP must explain how the author proposes to deal with these incompatibilities. MIP submissions without a sufficient backwards compatibility treatise may be rejected outright.
+- Test Cases - Test cases for an implementation are mandatory for MIPs that are affecting consensus changes. Other MIPs can choose to include links to test cases if applicable.
+- Implementations - The implementations must be completed before any MIP is given status “Final”, but it need not be completed before the MIP is merged as draft. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of “rough consensus and running code” is still useful when it comes to resolving many discussions of API details.
+- Copyright Waiver - All MIPs must be in the public domain. See the bottom of this MIP for an example copyright waiver.
+
+## MIP Formats and Templates
+
+MIPs should be written in [markdown] format.
+Image files should be included in a subdirectory of the `assets` folder for that MIP as follow: `assets/mip-X` (for mip **X**). When linking to an image in the MIP, use relative links such as `../assets/mip-X/image.png`.
+
+## MIP Header Preamble
+
+Each MIP must begin with an RFC 822 style header preamble, preceded and followed by three hyphens (`---`). The headers must appear in the following order. Headers marked with "*" are optional and are described below. All other headers are required.
+
+` mip:` <MIP number> (this is determined by the MIP editor)
```

Comment:
This field can only be filled in after actually sending the MIP as a PR and editing this field afterwards, right?

Reply:
technically no because the desired convention is to have the PR number become the MIP number.
I will revise to make that clear.